en/docs/install-and-setup/setup/api-controller/managing-apis-in-kubernetes/resources/kustomize-config.yaml

```diff
@@ -0,0 +1,13 @@
+nameReference:
+- kind: ConfigMap
+  fieldSpecs:
+  - path: spec/swaggerConfigMapName
+    kind: API
+- kind: ConfigMap
+  fieldSpecs:
+  - path: spec/paramsValues
+    kind: API
+- kind: Secret
+  fieldSpecs:
+  - path: spec/certsValues
+    kind: API
```

Comment:
Shall we add a newline at the EOF?

Reply:
Fixed with 21f466ce5d99225a24a45351eae0997e87f8c2f5